kubejs/client_scripts/expert/item_modifiers/jei_hide_items.js

```diff
@@ -3,6 +3,8 @@ onEvent('jei.hide.items', (event) => {
         return;
     }
     let items = [
+        'advancedperipherals:me_bridge',
+        'advancedperipherals:colony_integrator',
```

Comment:
Please add these here instead: https://github.com/NillerMedDild/Enigmatica6/blob/master/kubejs/client_scripts/constants.js#L103
Those items are hidden regardless of mode